packages/snaps-controllers/src/test-utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const MOCK_BLOCK_NUMBER = '0xa70e75';
```

Comment:
Moved this to a separate file because Vitest was trying to import Node.js dependencies through `./test-utils/index.ts`.